pkg/io/tfio/locals.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package tfio generates a locals.tf file that sets up some of the tf variables
```

Comment:
```suggestion
```
This is not the right documentation for the package.
Better make it the documentation of `WriteLocals()`